032-MLOpsFromScratch/Coach/02-UnitTesting.md

```diff
@@ -0,0 +1,23 @@
+# Challenge 2 – Create a Unit Test in Azure DevOps
+
+[< Previous Challenge](./01-TimeSeriesForecasting.md) - **[Home](../README.md)** - [Next Challenge >](./03-BuildPipeline.md)
+
+## Solution
+
+1.  If you haven’t already done this in
+    [Challenge\#1](01-TimeSeriesForecasting.md), create and setup a new project
+    in Azure DevOps
+
+    1.  Download and extract `Data_and_Code.zip` from Teams Channel 
+
+    2.  [Create new service connections](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml) in Project Settings for your Azure ML service and Azure Subscription using service

```

Comment:
Just like in the student text the reference to challenge 1 and the download install and service connection all feel redundant.

Reply:
Done